Lib/ast.py

```diff
@@ -582,6 +582,16 @@ def interleave(self, inter, f, seq):
                 inter()
                 f(x)
 
+    def items_view(self, traverser, items):
+        """If length of *items* is one, traverse item and end with comma
```

Comment:
This docstring is redundant: I can already read what the docstring is saying in the code. You should document what the result of the operation is, not how is achieved ;)

Reply:
> You should document what the result of the operation is, not how is achieved

Sure, I'll try.